pkgs/development/python-modules/swagger-ui-py/default.nix

```diff
@@ -0,0 +1,30 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, jinja2
+, pyyaml
+, lib
+}:
+
+buildPythonPackage rec {
+  pname = "swagger-ui-py";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "PWZER";
+    repo = "swagger-ui-py";
+    rev = "v${version}";
+    sha256 = "1910zvkp2613s4ysb61k2gkaw3m1hdg2433kbqil80mqm915i1ix";
+  };
+
+  propagatedBuildInputs = [
+    jinja2
+    pyyaml
+  ];
+
+  meta = with lib; {
+      description = "Swagger UI for Python web framework, such as Tornado, Flask, Quart, aiohttp, Sanic and Falcon.";
```

Comment:
Descriptions normally shouldn't start with the package name or end with a dot.

Reply:
Changed and noted.